scanpipe/tests/data/is-npm-1.0.0_scan_package.json

```diff
@@ -158,76 +161,7 @@
         "count": 1
       }
     ],
-    "packages": [
-      {
-        "type": "npm",
-        "namespace": null,
-        "name": "is-npm",
-        "version": "1.0.0",
-        "qualifiers": {},
-        "subpath": null,
-        "primary_language": "JavaScript",
-        "description": "Check if your code is running as an npm script",
-        "release_date": null,
-        "parties": [
-          {
-            "type": "person",
-            "role": "author",
-            "name": "Sindre Sorhus",
-            "email": "sindresorhus@gmail.com",
-            "url": "http://sindresorhus.com"
-          }
-        ],
-        "keywords": [
-          "npm",
-          "is",
-          "check",
-          "detect",
-          "env",
-          "environment"
-        ],
-        "homepage_url": null,
-        "download_url": "https://registry.npmjs.org/is-npm/-/is-npm-1.0.0.tgz",
-        "size": null,
-        "sha1": null,
-        "md5": null,
-        "sha256": null,
-        "sha512": null,
-        "bug_tracking_url": null,
-        "code_view_url": null,
-        "vcs_url": "https://github.com/sindresorhus/is-npm",
-        "copyright": null,
-        "license_expression": "mit",
-        "declared_license": [
-          "MIT"
-        ],
-        "notice_text": null,
-        "root_path": "codebase/package",
-        "dependencies": [
-          {
-            "purl": "pkg:npm/ava",
-            "requirement": "0.0.3",
-            "scope": "devDependencies",
-            "is_runtime": false,
-            "is_optional": true,
-            "is_resolved": false
-          }
-        ],
-        "contains_source_code": null,
-        "source_packages": [],
-        "extra_data": {},
-        "purl": "pkg:npm/is-npm@1.0.0",
-        "repository_homepage_url": "https://www.npmjs.com/package/is-npm",
-        "repository_download_url": "https://registry.npmjs.org/is-npm/-/is-npm-1.0.0.tgz",
-        "api_data_url": "https://registry.npmjs.org/is-npm/1.0.0",
-        "files": [
-          {
-            "path": "codebase/package/package.json",
-            "type": "file"
-          }
-        ]
-      }
-    ]
+    "packages": []
```

Comment:
@pombredanne any reasons why this package is not detected anymore?

Reply:
this is a serious regression.